src/main/java/org/folio/inventory/domain/items/ItemsMove.java

```diff
@@ -0,0 +1,35 @@
+package org.folio.inventory.domain.items;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ItemsMove {
+
+  public static final String TO_HOLDINGS_RECORD_ID = "toHoldingsRecordId";
+  public static final String ITEM_IDS = "itemIds";
+
+  private String toHoldingsRecordId;
+  private List<String> itemIds = new ArrayList<>();
+
+  public ItemsMove() {
+    super();
+  }
```

Comment:
Can it be removed?

Reply:
This was created to prevent code smell for empty public constructor since I use Builder. OK I can use parameterized constructor here instead of empty one with builders.

Reply:
ItemsMove is removed at all.